roles/lib_utils/filter_plugins/oo_filters.py

```diff
@@ -660,6 +660,39 @@ def map_from_pairs(source, delim="="):
     return dict(item.split(delim) for item in source.split(","))
 
 
+def lib_utils_oo_remove_undefined(source):
+    ''' Replace ansible Undefined value with empty dict '''
+    result = list()
+    for item in source:
+        if item:
+            result.append(item)
+    return result
+
+
+def lib_utils_oo_has_no_matching_selector(source, selector=None):
+    ''' Return True when selector cannot be placed
+        on nodes with labels from source '''
+    if not isinstance(selector, dict):
+        return True
+    for item in source:
+        if selector.items() <= item.items():
+            # Matching selector found
+            return False
+    return True
+
+
+def lib_utils_oo_get_schedulable_nodes(source, hostvars=""):
+    ''' Filter out nodes which are not schedulable '''
+    result = list()
+    for item in source:
+        if item not in hostvars:
+            return
+        item_hostvars = hostvars[item]
+        if item_hostvars.get('openshift_schedulable'):
```

Comment:
openshift_schedulable is not always defined.

openshift.node.schedulable is set by openshift_facts, default true for nodes, default false for masters.